pyproject.toml

```diff
@@ -0,0 +1,36 @@
+[tool.black]
+line-length = 120
+target_version = ['py27', 'py34', 'py35', 'py36', 'py37', 'py38']
+exclude = '''
```

Comment:
might be easier to just pass the `ddtrace` and `tests` directory :)

Reply:
yeah, I wanted to be exhaustive a bit here so if someone wanted to go in and remove one and slowly blackify the library they could without having to go through the effort of making up a crazy regex 😆